node.gyp

```diff
@@ -217,6 +224,37 @@
 
 
       'conditions': [
+        [ 'node_shared=="false"', {
+          'msvs_settings': {
+            'VCManifestTool': {
+              'EmbedManifest': 'true',
+              'AdditionalManifestFiles': 'src/res/node.exe.extra.manifest'
+            }
+          },
+        }, {
+          'defines': [
+            'NODE_SHARED_MODE',
+          ],
+          'conditions': [
+            [ 'node_module_ver!=""', {
+              'product_extension': 'so.<(node_module_ver)',
+            }]
+          ],
+        }],
+        [ 'node_use_bundled_v8=="true"', {
+          'include_dirs': [
+            'deps/v8', # include/v8_platform.h
+          ],
+
+          'dependencies': [
+            'deps/v8/tools/gyp/v8.gyp:v8',
+            'deps/v8/tools/gyp/v8.gyp:v8_libplatform'
+          ],
+        }, {
+          'defines': [
+            'NODE_NO_V8_PLATFORM',
```

Comment:
Can you call this `NODE_USE_V8_PLATFORM` and set it to 0 or 1 depending on whether `node_use_v8_platform` is true.

(If you make an int, you could shorten it to `'defines': [ 'NODE_USE_V8_PLATFORM=<(node_use_v8_platform)' ]`, although that would be inconsistent with the other node_ variables.)